.github/workflows/tests.yml

```diff
@@ -32,9 +32,12 @@ jobs:
             configure_options: "--with-petsc
                                 --with-slepc
                                 --with-sundials=/home/runner/local"
-            script_flags: "-uim"
+            cmake_options: "-DBOUT_USE_PETSC=ON
```

Comment:
For the `default options` build, we should not specify any options really -- at least partly because we really need a build without PETSc/SLEPc